ResearchKit/Common/ORKReviewStep.md

```diff
@@ -0,0 +1,81 @@
+<h2>Introduce</h2>
+- `ORKReviewStep`
+- `ORKReviewStepViewController`
+- `ORKReviewStepViewControllerDelegate`
+- `ORKReviewStepReviewDirection`
+
+<h2>Change</h2>
+- `ORKTaskViewController`
+
+<h2>ORKReviewStep</h2>
+<h3>properties</h3>
+    NSArray *steps
+    ORKTaskResultSource *resultSource
+    ORKReviewStepReviewDirection *reviewDirection
+<h3>use cases</h3>
+- __outside of a survey = standalone__
+  - steps array is provided upon initialization
+  - review step is at the beginning
+  - step results may not be changed
+  - review direction defaults to forward
+		
+- __within a survey__
+  - steps array is nil
+  - review step is at the end or in the middle
+  - step results can be changed
+  - review direction defaults to reverse			
+
+<h2>ORKReviewStepViewControllerDelegate</h2>
+<h3>methods</h3>
+    reviewStepViewController: canReviewStep: 
+        called to determine if a step can be reviewed
+    
+    reviewStepViewController: canChangeStep: 
+        called to determine if an answer can be changed during review
```

Comment:
We won't need `canChangeStep` anymore, if we add a similar property to `ORKStepViewController` (see my comment below).

`canReviewStep` asks the delegate if a step that was discovered by the `ORKReviewStepViewController` should actually be presented for review. This could be used to just display steps that have not been answered yet. To do so an app would need to subclass `ORKTaskViewController` and override the default implementation of `canReviewStep`.